### ISSUE TITLE
Editor like TinyMCE or CJE not loaded with IPv6-Adress

### ISSUE THREAD

**Issue by Kuurusch** [joomla/joomla-cms#8708]
#### Steps to reproduce the issue
When you load the administrator-Site with a raw IPv6-Adress, it's not possible to write or edit an article. The editor is not loaded and where it should be is only a white surface.
When I load the same administrator-Site with a normel internetadress (like www.testpage.org), everything works fine. 
=> So there is somehow a problem with the url to load the editor.

#### Expected result
There should be an editor in the edit-mode of an article.

#### Actual result
The editor is not loaded in the edit-mode of an article and where it should be is only a white surface.

#### System information (as much as possible)
It's a server in the new hyperboria network where at the moment it's quite common to use the raw IPv6-adress.

#### Additional comments
It would be nice to make Joomla IPv6-able.

**Comment by brianteeman** [comment]
You can use MAMP to test IPv6 addresses.

I can confirm that NO WYSIWYG editor loads when using IPv6 numeric address

**Comment by brianteeman** [comment]
Looking into this further at tinymce there is an old report of this
http://archive.tinymce.com/develop/bugtracker_view.php?id=2708

Checking the unit tests for the referred code block we can see that tinymce
is not testing agains numeric ipv6 addresses

Not sure if this is something we can fix maybe @dgt41 has some ideas

On 16 December 2015 at 12:40, Brian Teeman <brian@teeman.net> wrote:

> You can use MAMP to test IPv6 addresses.
>
> I can confirm that NO WYSIWYG editor loads when using IPv6 numeric address
> On 16 Dec 2015 12:18 pm, "Kuurusch" <notifications@github.com> wrote:
>
>> Steps to reproduce the issue
>>
>> When you load the administrator-Site with a raw IPv6-Adress, it's not
>> possible to write or edit an article The editor is not loaded and where it
>> should be is only a white surface
>> When I load the same administrator-Site with a normel internetadress
>> (like wwwtestpageorg), everything works fine
>> => So there is somehow a problem with the url to load the editor
>> Expected result
>>
>> There should be an editor in the edit-mode of an article
>> Actual result
>>
>> The editor is not loaded in the edit-mode of an article and where it
>> should be is only a white surface
>> System information (as much as possible)
>>
>> It's a server in the new hyperboria network where at the moment it's
>> quite common to use the raw IPv6-adress
>> Additional comments
>>
>> It would be nice to make Joomla IPv6-able
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/joomla/joomla-cms/issues/8708>.
>>
>


-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by dgt41** [comment]
I haven't try this but it might solve the problem,
so after line 717 insert:
```php
		tinyMCE.suffix = '.min';
		tinyMCE.baseURL = JUri::root(true) . '/media/editors/tinymce';
```

**Comment by brianteeman** [comment]
I will test soon

**Comment by brianteeman** [comment]
Either I put the code in the wrong place or it doesnt help
After adding the code tinymce only loads in Simple mode and no change was
observed with ipv6

On 16 December 2015 at 13:30, Brian Teeman <brian@teeman.net> wrote:

> I will test soon
> On 16 Dec 2015 1:22 pm, "Dimitri Grammatikogianni" <
> notifications@github.com> wrote:
>
>> I haven't try this but it might solve the problem,
>> so after line 717 insert:
>>
>>         tinyMCE.suffix = '.min';        tinyMCE.baseURL = JUri::root(true) . '/media/editors/tinymce';
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/joomla/joomla-cms/issues/8708#issuecomment-165102871>
>> .
>>
>


-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by dgt41** [comment]
Sorry the code was wrong, should be:
```php
				tinyMCE.suffix = '.min';
		tinyMCE.baseURL = '" . JUri::root() . "media/editors/tinymce';
		tinymce.init({
```
replacing the line 718: `tinymce.init({`
I get successful loading of tinyMCE but with a css file loading error (or something like that:
![screen shot 2015-12-16 at 16 08 07](https://cloud.githubusercontent.com/assets/3889375/11842965/932b5c72-a40f-11e5-8c0c-a866bd1fa003.png)

**Comment by brianteeman** [comment]
That works for me - I dont see the console error (but that could be me)

On 16 December 2015 at 14:11, Dimitri Grammatikogianni <
notifications@github.com> wrote:

> Sorry the code was wrong, should be:
>
>                 tinyMCE.suffix = '.min';        tinyMCE.baseURL = '" . JUri::root() . "media/editors/tinymce';        tinymce.init({
>
> replacing the line 718: tinymce.init({
> I get successful loading of tinyMCE but with a css file loading error (or
> something like that:
> [image: screen shot 2015-12-16 at 16 08 07]
> <https://cloud.githubusercontent.com/assets/3889375/11842965/932b5c72-a40f-11e5-8c0c-a866bd1fa003.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/issues/8708#issuecomment-165117365>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by dgt41** [comment]
For some stupid reason when I closed https://github.com/joomla/joomla-cms/pull/5839 i forgot to bring all those changes to 3.5. This is essential not only for IPV6 but also for any plugin/template that concatenate the javascripts...

**Comment by brianteeman** [comment]
I will leave it to you to do the PR then ;)

On 16 December 2015 at 14:27, Dimitri Grammatikogianni <
notifications@github.com> wrote:

> For some stupid reason when I closed #5839
> <https://github.com/joomla/joomla-cms/pull/5839> i forgot to bring all
> those changes to 3.5. This is essential not only for IPV6 but also for any
> plugin/template that concatenate the javascripts...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/issues/8708#issuecomment-165124922>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by brianteeman** [comment]
Closed as we have a PR for testing #8711 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8708">issues.joomla.org/joomla-cms/8708</a>.</sub>
